app/src/main/res/values/dimens.xml

```diff
@@ -32,6 +32,7 @@
     <!-- Elevation -->
     <dimen name="elevation_2dp">2dp</dimen>
     <dimen name="elevation_4dp">4dp</dimen>
+    <dimen name="elevation_touch_raise">6dp</dimen>
```

Comment:
NP.
But always touch_raise is always 6dp. So I was slightly troubled.
https://material.io/guidelines/material-design/elevation-shadows.html#elevation-shadows-elevation-android